Checkout/Checkout/Test/Model/CountryTests.swift

```diff
@@ -245,7 +245,7 @@ final class CountryTests: XCTestCase {
       Country(iso3166Alpha2: "AE", dialingCode: "971"),
       Country(iso3166Alpha2: "GB", dialingCode: "44"),
       Country(iso3166Alpha2: "US", dialingCode: "1"),
-      Country(iso3166Alpha2: "UMI", dialingCode: "246"),
+      Country(iso3166Alpha2: "UM", dialingCode: "246"),
```

Comment:
how to know that `diallingCode` is the right one for` iso3166Alpha2`? is there any unit test for it ?